mavros_extras/msg/CamIMUStamp.msg

```diff
@@ -0,0 +1,4 @@
+# IMU-Camera synchronisation data
```

Comment:
No need here. This message will be parsed by whatever camera driver the user is using, sequence ID matched and the image timestamp set to timestamp from this message :)